README.md

```diff
@@ -40,9 +40,9 @@ val helloWorldCounter = Counter("hello-world-counter")
 
 def printAndIncrementMetric[F[_] : Sync : IncrementMeric[?[_], Counter]] : F[Unit] =
   for {
-    _ <- IncrementMetric[F, Counter].increment(helloWorldCounter.refine("status" -> "before"))
+    _ <- IncrementMetric[F, Counter].increment(helloWorldCounter.withTag("status" -> "before"))
     _ <- Sync[F].delay { println("Hello world" }
-    _ <- IncrementMetric[F, Counter].increment(helloWorldCounter.refine("status" -> "after"))
+    _ <- IncrementMetric[F, Counter].increment(helloWorldCounter.withTag("status" -> "after"))
```

Comment:
Due to the API changes to `withTag`, I think this example should now read `withTag("status", "before")` and `withTag("status", "after")`.

Aside from this comment, all of the other changes LGTM.

Reply:
Good catch.